ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/keys/ContractKeysValidation.scala

```diff
@@ -65,21 +66,19 @@ private[transaction] object ContractKeysValidation {
       contractKeysToContractIds: Map[DamlContractKey, RawContractId],
       transactionEntry: DamlTransactionEntrySummary,
   ): StepResult[DamlTransactionEntrySummary] = {
+
     val keysValidationOutcome = transactionEntry.transaction
       .foldInExecutionOrder[KeyValidationStatus](
-        Right(
-          KeyValidationState(activeStateKeys = contractKeyDamlStateKeys)
-        )
+        Right(KeyValidationState(activeStateKeys = contractKeyDamlStateKeys))
       )(
-        (keyValidationStatus, _, exerciseBeginNode) =>
-          checkNodeContractKey(
-            exerciseBeginNode,
-            contractKeysToContractIds,
-            keyValidationStatus,
-          ),
-        (keyValidationStatus, _, leafNode) =>
-          checkNodeContractKey(leafNode, contractKeysToContractIds, keyValidationStatus),
-        (accum, _, _) => accum,
+        exerciseBegin = (status, _, exerciseBeginNode) => {
+          onCurrentStatus(status)(
+            checkNodeContractKey(exerciseBeginNode, contractKeysToContractIds, _)
+          )
+        },
```

Comment:
I believe the curly braces can be removed; the newlines also perhaps, as it seems roughly the same line length as below to me.

Reply:
I wish scalafmt would just do this for me cff0b3d04370fe63bf24b9bd93cf79edac9c5b51